docs/modules/ROOT/pages/how-tos/schedules.adoc

```diff
@@ -0,0 +1,16 @@
+= How to Create a Schedule
+
+The `Schedule` object defines the frequency, destination and secrets required to run K8up jobs in your namespace:
+
+[source,yaml]
+----
+include::example$schedule.yaml[]
+----
+
+Save the YAML above in a file named `schedule.yaml` and use the `kubectl apply -f schedule.yaml` command to deploy this configuration to your cluster.
+
+The file above will instruct the Operator to do backups every 5 minutes and a monthly prune, and check jobs for repository maintenance. It will also archive the latest snapshots to the `archive` bucket once each week.
```

Comment:
```suggestion
The file above will instruct the Operator to do backups every 5 minute, prune them monthly and run check jobs for repository maintenance. It will also archive the latest snapshots to the `archive` bucket once each week.
```